addons/mail/static/src/js/composer.js

```diff
@@ -469,7 +469,16 @@ var BasicComposer = Widget.extend(chat_mixin, {
     preprocess_message: function () {
         // Return a deferred as this function is extended with asynchronous
         // behavior for the chatter composer
-        var value = _.escape(this.$input.val()).replace(/\n|\r/g, '<br/>');
+
+        //Remove Extra newlines from starting and ending of Message.
+        var value = _.escape(this.$input.val()).replace(/^(\r|\n)+|(\n|\r)+$/g, '');
```

Comment:
Maybe use `_.string.trim` instead of reimplementing it by hand? Though beware that trim() removes all leading an trailing whitespace, I do not know that this is actually and exactly what we want.